modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb

```diff
@@ -25,6 +25,10 @@ def upload(claim_id)
                              "Successfully submitted to Docker container with response: #{evss_res}")
         # update with the evss_id returned
         auto_claim.update!(evss_id: evss_res[:claimId])
+      rescue => e
+        auto_claim.evss_response = e&.errors if e.methods.include?(:errors)
```

Comment:
nit: Do we need `&` here?

`e` should always be defined, and even if it weren't, it would fail at the `e.methods.include?` conditional before reaching `e&.errors`.

Reply:
@stiehlrod  One thing to consider with this.  It may be easier once Matts PR removing the extra 'service' file goes in.  Then we could use the shared methods to handle a lot of this.  

We could set the errored state and save the evss_response something like 

````
        set_errored_state_on_claim(auto_claim)
        set_evss_response(auto_claim, e)
```
(sudo code ^)